Bottom: 42a124064cbe96585b4e76f2f802bd5a7cb7358b
Top:    f7c2b2dc7a5a1dd627be0e2d0ca7b9ffa9521d72
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 02:16:50 +0100

crosshair.c: Nasty thing FIXME


---

diff --git a/src/crosshair.c b/src/crosshair.c
index ca51f4f..f10a250 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -692,8 +692,10 @@ CrosshairOn (void)
   if (!Crosshair.On)
     {
       Crosshair.On = true;
+#if 0
       DrawAttached ();
       DrawMark ();
+#endif
     }
 }
 
@@ -706,8 +708,10 @@ CrosshairOff (void)
   if (Crosshair.On)
     {
       Crosshair.On = false;
+#if 0
       DrawAttached ();
       DrawMark ();
+#endif
     }
 }
